develop/dev-guide-third-party-support.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Third-Party Libraries Supported by PingCAP
+summary: Learn about third-party libraries supported by PingCAP.
+---
+
+# Third-Party Libraries Supported by PingCAP
+
+TiDB's support for the MySQL protocol makes most of the MySQL drivers, ORM frameworks, and other tools that adapt to MySQL compatible with TiDB. This document focuses on these tools and their support levels.
+
+## Support Level
+
+PingCAP works with community and provides the following levels of support:
+
+- **_Full_**: Indicates that PingCAP is already compatible with most of the tool's functionalities, and maintains compatibility with its newer versions. PingCAP will periodically conduct compatibility tests with the latest version of third-party tools documented in the table below.
+- **_Compatible_**: Indicates that because the tool is adapted to MySQL, and TiDB is highly compatible with the MySQL protocol. So it can use most of the tool's features. However, PingCAP has not complete test this tool's features. It may lead to some unexpected behavior.
```

Comment:
```suggestion
- **_Compatible_**: Indicates that because the corresponding third-party tool is adapted to MySQL and TiDB is highly compatible with the MySQL protocol, so TiDB can use most features of the tool. However, PingCAP has not completed a full test on all features of the tool, which might lead to some unexpected behaviors.
```